### PR TITLE
Add support for targets

### PR DESCRIPTION
**Pull request by onematchfox**

### CODE REVIEW

**Comment by onematchfox** [comment]
Resolves outstanding points on https://github.com/kevholditch/terraform-provider-kong/issues/52

File: README.md [hunk 1]
```diff
@@ -310,6 +310,31 @@ resource "kong_upstream" "upstream" {
     slots 		= 10
 }
 ```
+`name` is a hostname like name that can be referenced in the upstream_url field of a service.
+`slots` is the number of slots in the load balancer algorithm (10-65536, defaults to 1000).
```
Comment (kevholditch) [inline README.md]:
```suggestion
`slots` is the number of slots in the load balancer algorithm (10-65536, defaults to 10000).
```

File: kong/resource_kong_target_test.go [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+const testCreateTargetConfig = `
+resource "kong_upstream" "upstream" {
+	name  		= "MyUpstream"
+	slots 		= 10
+}
+
+resource "kong_target" "target" {
+	target 		  = "mytarget:4000"
+	weight 		  = 100
+  upstream_id = "${kong_upstream.upstream.id}"
```
Comment (kevholditch) [inline kong/resource_kong_target_test.go]: can you please allign `upstream_id` with the column above

File: kong/resource_kong_target_test.go [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+}
+`
+const testUpdateTargetConfig = `
+resource "kong_upstream" "upstream" {
+	name  		= "MyUpstream"
+	slots 		= 10
+}
+
+resource "kong_target" "target" {
+  target 	  	= "mytarget:4000"
```
Comment (kevholditch) [inline kong/resource_kong_target_test.go]: can you allign with row below

File: kong/resource_kong_target_test.go [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+	weight 	   	= 200
+  upstream_id = "${kong_upstream.upstream.id}"
```
Comment (kevholditch) [inline kong/resource_kong_target_test.go]: can you allign with above

File: kong/resource_kong_target_test.go (hunk 2 above)
Comment (onematchfox) [reply kong/resource_kong_target_test.go]: All done. Sorry... tab vs space issue in editor.

**Comment by kevholditch** [comment]
@onematchfox thanks for your contribution, your changes will be in version `v4.1.0` of the provider

**Pull request merged**